node_modules/gh-core/lib/globals/sequelize.js

```diff
@@ -27,18 +37,21 @@ var Promise = require('sequelize/lib/promise');
  */
 Promise.prototype.complete = function(callback) {
     this
+
+        // Failure condition. This must be registered before the `then` condition to ensure that an
+        // error in the callback does not get caught by this handler, risking that the callback be
+        // invoked twice
+        .catch(function(err) {
+            // Send the err argument to the consumer as the first and only parameter
+            callback.call(null, err);
+        })
+
         // Success condition
         .then(function() {
             // Shift a null value as the first parameter in the callback to indicate the result is
             // a success
             var args = Array.prototype.slice.call(arguments);
             args.unshift(null);
-            return callback.apply(null, args);
-        })
-
-        // Failure condition
-        .catch(function(err) {
-            // Send the err argument to the consumer as the first and only parameter
-            callback.call(null, err);
+            callback.apply(null, args);
```

Comment:
Note, I took away the returns for this because `return`'s actually mean something in promises :P